### ISSUE TITLE
Game crashing on startup

### ISSUE THREAD

**Issue by wizard23000** [DarkPacks/SevTech-Ages#2296]
## Context
Trying to play this modpack, and I can't really play it when it crashes on startup

## Logs
I do not know where to find my "crash-logs" folder, as these are the only things I can find in my SevTechAges folder, otherwise I would post it
![sevtechinfo](https://user-images.githubusercontent.com/24814769/39842596-a2a0936c-53ac-11e8-91b0-5d2148c12b23.png)

## Client Information
* Modpack Version: 3.0.6
* Java Version: Version 8 update 171
* Launcher Used: ATLauncher
* Memory Allocated: 5.0 GB

**Comment by wizard23000** [comment]
I tried changing to the Twitch launcher, and updating it to 3.0.7, and it still did not fix the issue, and getting rid of BetterFPS, as it seems that it was a mod causing ATLauncher to crash, once I get the update available on ATLauncher, I will see if the update helps, but I doubt it will

**Comment by wizard23000** [comment]
I have also recorded what happens when I try to launch the game from both launchers: https://youtu.be/m-93gl5OX_U

**Comment by artdude543** [comment]
Can you at least paste bin your client log like the template asks? No Logs = No Help

**Comment by wizard23000** [comment]
I think this is what you were looking for, honestly never looked through my logs before now so idk: https://pastebin.com/muxG7zJs

**Comment by shado-wisper** [comment]
Issue / Bug : Game crash

Context: Updated SevTech Ages to 3.0.7 on the Twitch desktop app and attempted to play, but it showed a generic Minecraft Java edition screen after the usual loading screen, and crashed when I clicked my single player game. Also noticed there's only 8 installed mods after the update. Previously played SevTech Ages 3.0.6 with no problems.

Client Information
    Modpack Version: SevTech Ages 3.0.7
    Server/LAN/Single Player: Single Player
    Resourcepack Enabled?: Yes, Soartex_Fanver_Modded_Universal 4.1.2
    Additional Content Installed?: No

System Information
    Java Version: 1.8.0_161
    Launcher Used: Twitch-Jar Launcher/Minecraft launcher 1.6.76
    Operating System: Windows 10
    Memory Allocated: 5120MB out of 24GB

Logs:
[crash-2018-05-09_20.11.11-server.txt](https://github.com/DarkPacks/SevTech-Ages/files/1989965/crash-2018-05-09_20.11.11-server.txt)
